_data/settings.yml

```diff
@@ -24,96 +24,96 @@ linaro-footer-image: Linaro-Sprinkle.png
 #Content Repo for static site.
 # Used for the edit on github include
 edit-on-github:
-    enabled: true
-    repo: https://www.github.com/Linaro/connect
+  enabled: true
+  repo: https://www.github.com/Linaro/connect
 # Settings for the home page
 home:
-    #Content Background Image E.g. DeviceTree.org
-    # content-image: background-image.png
-    # If this is set then the navigation will be transparent.
-    transparent-nav: true
-    # If this is set to true then the shape divider will be displayed
-    shaped-divider: false
-    # Jumbotron settings for the home page
-    jumbotron:
-        # Include file to replace all the contents of the jumbotron with.
-        included_file: hero-banner.html
-        # Toggle the jumbotron
-        enabled: true
-        # Background image for the home page located in assets/images
-        background-image:
-            # Darken is set to true by default - if you wish to remove the image overlay
-            # then change this setting to false.
-            darken: true
-            # This is the name of the background-image located in /assets/images/
-            name: bkk19-website-banner.jpg
+  #Content Background Image E.g. DeviceTree.org
+  # content-image: background-image.png
+  # If this is set then the navigation will be transparent.
+  transparent-nav: true
+  # If this is set to true then the shape divider will be displayed
+  shaped-divider: false
+  # Jumbotron settings for the home page
+  jumbotron:
+    # Include file to replace all the contents of the jumbotron with.
+    included_file: hero-banner.html
+    # Toggle the jumbotron
+    enabled: true
+    # Background image for the home page located in assets/images
+    background-image:
+      # Darken is set to true by default - if you wish to remove the image overlay
+      # then change this setting to false.
+      darken: true
+      # This is the name of the background-image located in /assets/images/
+      name: bkk19-website-banner.jpg
 # Jekyll social media site settings. Github, LinkedIn, Google+, Facebook and Youtube
 github_username: Linaro
 linkedin_username: 1026961
-google_plus_username: +LinaroOnAir
+google_plus_username: false
```

Comment:
@kylekirkby , can this just be removed now? Google+ was shutdown for general use on [2019-04-02](https://support.google.com/plus/?hl=en#topic=9259565). I don't see this key managed on [jumbo-jekyll-theme](https://github.com/linaro-marketing/jumbo-jekyll-theme).